docs/getting-started.md

```diff
@@ -0,0 +1,88 @@
+# Getting Started
+
+## Installation
+
+Install `kitbag/router` with your favorite package manager
+
+```bash
+# bun
+bun add kitbag/router
+# yarn
+yarn add kitbag/router
+# npm
+npm install kitbag/router
+```
+
+## Define Basic Routes
+
+Create an array of possible routes. Learn more about [defining routes](/core-concepts/defining-routes).
+
+```js
+// /routes.ts
+import { Routes } from '@kitbag/router'
+
+const Home = { template: '<div>Home</div>' }
+const About = { template: '<div>About</div>' }
+
+export const routes = [
+  { name: 'home', path: '/', component: Home },
+  { name: 'path', path: '/about', component: About },
+] as const satisfies Routes 
+```
+
+## Plugin
+
+Create a router instance and pass it to the app as a plugin
+
+```js {2-3,6,9}
+import { createApp } from 'vue'
+import { createRouter } from '@kitbag/router'
+import { routes } from '/routes'
+import App from './App.vue'
+
+const router = createRouter(routes)
+const app = createApp(App)
+
+app.use(router)
+app.mount('#app')
+```
+
+## Update Registered Router
+
+This block utilizes [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) to provide the internal types to match the actual router you're using. You put this in main.ts right after you call `createRouter`, or you can export your router and put this interface inside of a `router.d.ts` file, anywhere that your tsconfig can find it.
+
+```ts
+declare module '@kitbag/router' {
+  interface Register {
+    router: typeof router
+  }
+}
+```
+
+## RouterView
+
+Give your route components a place to be mounted
+
+```html {3-4}
+<div id="app">
```

Comment:
Assuming this is the `App.vue` components I don't think it would have `id="app"`. That would be in your main html file that the vue app is mounted to.

Reply:
I did [what vue-router did](https://router.vuejs.org/guide/#HTML), but I agree with you. I'll move from id= to class=? encourage BEM lol